### ISSUE TITLE
Temporary Homepage for Spring/Summer 2020

### ISSUE THREAD

**Issue by cchristopherhowardd** [aarome/aarome#270]
If the new website launches on April 24, we will need to modify the homepage from April to September, replacing the Events slider at the top of the page. The Academy has canceled all public programming for the academic year, so everything that's in the slider now (McKim Medal Gala, various Shoptalks, etc.) is obsolete. We may need to replace the slideshow with a single item that announces the 2020 Rome Prize winners--at least for the April 24 launch and through the month of May.

I anticipate another homepage complication: the slideshow of Fellows and Italian Fellows and the slideshow of Residents. These are all current-year people (now sent home), which may confuse website visitors who understand them to be the new crop of Rome Prize winners.

**Comment by jasloe** [comment]
Not a problem. I have a couple of new ideas for the slideshow to reflect our circumstances. I'll test them out on dev this week.

**Comment by cchristopherhowardd** [comment]
An idea I had for the homepage slider is to have five or six of the most recent items in Quick News ( https://staging.aarome.org/index.php/news/quick-news ) appear here, since I am adding things to this section once or twice a week. The problem is, links to each item take our visitors off the AAR website. A "fix" would be to have each link open in a new window, but there's still the possibility that visitors won't return to AAR.

**Comment by cchristopherhowardd** [comment]
Wow, that's great. 

Is there any chance the homepage could be updated today? I'm asking for a friend who would like to present it (probably as a screenshot in a PowerPoint presentation on Zoom) to the AAR Board of Trustees tomorrow.

**Comment by jasloe** [comment]
Sure, I can switch gears and get that done today.

**Comment by jasloe** [comment]
Had a couple of setbacks this afternoon with another project. Will be able to wrap it up by this evening. I'll let you know when changes are posted and I'll also attach screenshots here.

**Comment by jasloe** [comment]
@cchristopherhowardd –– Pushed to dev. This uses the old slideshow library that has a couple of bugs on window resizing and linking to inside content. Will have those resolved very soon. I think there are one or two things that need Open refinements. 

![Screen Shot 2020-06-12 at 8 11 34 AM](https://user-images.githubusercontent.com/1063667/84501616-ca6ca600-ac84-11ea-96c9-80d59665a8c8.png)
![Screen Shot 2020-06-12 at 8 12 04 AM](https://user-images.githubusercontent.com/1063667/84501628-ce002d00-ac84-11ea-8648-ba1806d787a1.png)
![Screen Shot 2020-06-12 at 8 12 23 AM](https://user-images.githubusercontent.com/1063667/84501635-d0628700-ac84-11ea-912c-f2442079141f.png)

**Comment by cchristopherhowardd** [comment]
Thanks. I think that Marques was able to use this link (or these images) for his presentation.

**Comment by cchristopherhowardd** [comment]
The page looks great. The only change I suggest is removing the date line from the slideshow. We can just have image and headline.

**Comment by cchristopherhowardd** [comment]
I've published People pages for the 2021 Rome Prize Fellows, which means we can update the "Meet the Fellows" part of the homepage. I don't know if the 2021 Italian Fellows have been chosen, so the slideshow does not need to draw from any Italian Fellows people pages.

We decided to not temporary unpublish the Invited Residents. It might be nice, though, to have this homepage section draw from People pages for Residents for both 2020 and 2019, so that there's a fuller pool of photos.
